api/lib/models/project/project-data-mapper-impl.ts

```diff
@@ -208,11 +208,13 @@ export class ProjectDataMapperImpl extends GenericDataMapper
       this.logger.warn(object.dbRepresentation);
       return Promise.reject({ result: 'error', data: new HttpError(validation.error, 400) });
     }
-    const query = squel.update({ autoQuoteFieldNames: true, autoQuoteTableNames: true })
+    let queryBuilder = squel.update({ autoQuoteFieldNames: true, autoQuoteTableNames: true })
       .table(this.tableName)
-      .setFields(object.dbRepresentation)
-      .where(`${this.pkColumnName} = ?`, object.id)
-      .toParam();
+      .where(`${this.pkColumnName} = ?`, object.id);
+    if (object.team) {
+      queryBuilder = queryBuilder.set('team', object.team.join(','));
+    }
```

Comment:
Do the same for object.categories, since that is also an array. And since rest of the fields aren't getting set otherwise, you'll have to include those too.